src/main/scala/ch/epfl/bluebrain/nexus/sbt/nexus/ServicePackagingPlugin.scala

```diff
@@ -0,0 +1,72 @@
+package ch.epfl.bluebrain.nexus.sbt.nexus
+
+import java.util.regex.Pattern
+
+import com.typesafe.sbt.packager.Keys._
+import com.typesafe.sbt.packager.archetypes.JavaAppPackaging
+import com.typesafe.sbt.packager.docker.DockerPlugin.autoImport.{Docker, DockerAlias}
+import com.typesafe.sbt.packager.docker.{Cmd, DockerPlugin, ExecCmd}
+import com.typesafe.sbt.packager.universal.UniversalPlugin
+import sbt.Keys._
+import sbt._
+
+/**
+  * Plugin defining default configuration for building docker images for akka based service implementations.
+  */
+object ServicePackagingPlugin extends AutoPlugin {
+
+  override lazy val requires = UniversalPlugin && JavaAppPackaging && DockerPlugin
+
+  override lazy val trigger = noTrigger
+
+  override lazy val projectSettings = Seq(
+    maintainer         := "Nexus Team <noreply@epfl.ch>",
+    dockerBaseImage    := "openjdk:8-jre",
+    daemonUser         := "root",
+    dockerExposedPorts := Seq(8080, 2552),
+    dockerRepository   := sys.env.get("DOCKER_REGISTRY"),
```

Comment:
This will fail during the build if we don't specify a registry; could we have a default value that gets overridden by the env value?